addons/mail/static/src/core/web/chatter.xml

```diff
@@ -17,7 +17,7 @@
                     'btn-primary active': state.composerType === 'note',
                     'btn-secondary': state.composerType !== 'note',
                     'my-2': !props.compactHeight
-                }" data-hotkey="shift+m" t-on-click="() => this.toggleComposer('note')">
+                }" t-att-disabled="!state.thread.hasWriteAccess and !(state.thread.hasReadAccess and state.thread.canPostOnReadonly) and props.threadId" data-hotkey="shift+m" t-on-click="() => this.toggleComposer('note')">
```

Comment:
Ok since they are assigned by hand in the important flow:
https://github.com/odoo/odoo/blob/17.0/addons/mail/static/src/core/web/thread_service_patch.js#L50

But theoretically the `hasReadAccess` and `canPostOnReadonly` should be added in `assignDefined` here:
https://github.com/odoo/odoo/blob/17.0/addons/mail/static/src/core/common/thread_model.js#L77 in case other flow assign using `.update()`. Note this is a cumbersome design specific to 17.0.